ecosystem/sep-0029.md

```diff
@@ -23,50 +23,50 @@ Users frequently forget to fill in a memo in their deposit transactions.
 from the custodial support staff.
 
-We introduce a basic convention that allows defining memo requirements on the
+We introduce a basic convention that allows defining memo requirements at the
 account level.
 
 Although this interoperability standard aims to deal with a missing memo problem
-in the first place, it can be also utilized by a wider class of Stellar-based
+in the first place, it can also be utilized by a wider class of Stellar-based
 applications to designate additional interaction or prerequisites.
 
 ## Abstract
 
 An account owner configures memo requirements by adding a data entry that
-specifies the whether a transaction memo is required when transferring funds to
+specifies whether a transaction memo is required when transferring funds to
 the account. A payment sender ensures that a memo is attached before submitting
 the transaction to the network.
 
 ## Specification
 
 To set up memo prerequisites for an account, the owner submits a transaction
-with `MANAGE_DATA` operation that adds a data entry with the name
-`config.memo_required` (in lower case following SEP-18 notation) and value `1`
+with a `MANAGE_DATA` operation that adds a data entry with the name
+`config.memo_required` (in lower case following [SEP-18](ecosystem/sep-0018.md) notation) and value `1`
 (a single character string "1" - ASCII code 49 or Unicode U+0031).
 
 When a payment sender submits a transaction containing a `PAYMENT`,
 `PATH_PAYMENT_STRICT_SEND`, `PATH_PAYMENT_STRICT_RECEIVE`, or `MERGE_ACCOUNT`
 operation they:
 
 - Load details about the destination account.
-- Checks if the destination account has `config.memo_required` data entry set.
+- Check if the destination account has the `config.memo_required` data entry set.
   If so:
-   - Verifies that the transaction `memo` field is set.
+   - Verify that the transaction `memo` field is set.
    - If `memo` is not set, shows a notification and prevents the transaction
      from submission.
 - For transactions containing more than one operation, perform the same checks
   for each operation.
 
 ## Rationale
 
-Multiplexed accounts proposal by David Mazières is more advanced and
+The proposal [First-class multiplexed accounts (CAP-27)](core/cap-0027.md) is a more advanced and
 comprehensive solution for the problem described above, but its adoption by the
 whole ecosystem and custodial exchanges will require much more time, as well as
-development efforts. Specifying memo requirements in a form of data entry
-attached to the account introduces a simple way of enforcing transaction
+development efforts. Specifying memo requirements in the form of a data entry
+attached to an account introduces a simple way of enforcing transaction
 memo requirements without changing the underlying protocol. 
 
-The name of the data entry follows the namespaces format proposed in SEP-18.
+The name of the data entry follows the namespaces format proposed in [SEP-18](ecosystem/sep-0018.md).
```

Comment:
FYI you can do these references as `[CAP-27]` and `[SEP-18]` and then move the actual path/URL to the end of the document like below, so that you don't have to repeat the path/URL multiple times.

```markdown
[CAP-27]: (core/cap-0027.md)
[SEP-18]: (ecosystem/sep-0018.md)
```

Reply:
I knew that once